src/patterns/atoms/logo/logo.hbs

```diff
@@ -0,0 +1,21 @@
+---
+name: Logos
+description: |
+  This component uses an image replacement technique for better SEO. You can, of course, still use a regular img element to insert a logo if it fits your use case better. Don't forget the alt text.
+
+  1. Add the CSS files to your bundle. You will need both the file for logo and for the product logo you want.
+  2. Add the component class `mzp-c-logo` the size class (eg: `mzp-t-md`) and the product class (eg: `mzp-t-product-firefox`) to your page
+     - For vertical spacing put the logo classes on or inside a heading or paragraph.
+order: 1
+notes: |
+tips: |
+  - Logos need both a size and product theme to display.
+    - Size classes are: `mzp-t-xs`, `mzp-t-sm`, `mzp-t-md`, `mzp-t-lg`, `mzp-t-xl`.
```

Comment:
I've been using theme/variant classes that reference the component, such as `mzp-t-content-md`, I guess just to make it clearer what it's applied to but it might also avoid potential conflicts if we end up with something that gets two different theme classes. 

So these could be `mzp-t-logo-md` (or `mzp-t-wordmark-md`). The benefit of clarity might be worth the cost of verbosity.

Reply:
Other review comments fixed. Still thinking about how I feel about this one. I was following existing conventions from stuff like buttons but that's not a reason not to do something better. I'm currently avoiding conflicts by requiring them to be used with the component class but that doesn't mean explicit isn't better. I like the shortness and predictability of using `mzp-t-sm` everywhere you want a small thing but with gzip I'm not sure there's any performance benefits. Anyway, need to sleep on this again I guess.

Reply:
Yeah, I've gone back and forth too. I ended up doing longer classes on the content container because when it was just "mzp-t-md" that didn't feel descriptive enough, at least in that case... like "what's medium here?" since it was a layout wrapper containing lots of things, and not a single component. The more explicit name also matched up with the tokens so it felt right. But on buttons it's on a small, specific component so it's more obvious what the size applies to... and the same is true here.

But in general I think maybe explicit is better than implicit, as long as we can still keep it pretty short. But does that mean we should change the buttons to all be `mzp-t-button-md`? Maybe so... 

Naming things is hard.

Reply:
Updated the names.